specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/stable/2022-11-23/confidentialledger.json

```diff
@@ -0,0 +1,1021 @@
+        "roleName"
+      ],
+      "properties": {
+        "roleName": {
+          "$ref": "#/definitions/LedgerUserRole"
+        },
+        "description": {
+          "description": "Description of the role.",
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "Collection": {
+      "description": "Identifier for collections.",
+      "required": [
+        "collectionId"
+      ],
+      "properties": {
+        "collectionId": {
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "PagedCollections": {
+      "description": "Paginated collections returned in response to a query.",
+      "type": "object",
+      "required": [
+        "collections"
+      ],
+      "properties": {
+        "collections": {
+          "items": {
+            "$ref": "#/definitions/Collection"
+          },
+          "type": "array"
+        },
+        "nextLink": {
+          "description": "Path from which to retrieve the next page of results.",
+          "type": "string"
+        }
+      }
+    },
+    "TransactionId": {
+      "description": "A unique identifier for the state of the ledger. If returned as part of a LedgerEntry, it indicates the state from which the entry was read.",
+      "type": "string"
+    },
+    "TransactionReceipt": {
+      "description": "A receipt certifying the transaction at the specified id.",
+      "required": [
+        "state",
+        "transactionId"
+      ],
+      "properties": {
+        "applicationClaims": {
+          "$ref": "#/definitions/ApplicationClaims"
+        },
+        "receipt": {
+          "$ref": "#/definitions/ReceiptContents"
+        },
+        "state": {
+          "$ref": "#/definitions/LedgerQueryState"
+        },
+        "transactionId": {
+          "$ref": "#/definitions/TransactionId"
+        }
+      },
+      "type": "object"
+    },
+    "TransactionState": {
+      "type": "string",
+      "description": "Represents the state of the transaction.",
+      "enum": [
+        "Committed",
+        "Pending"
+      ],
+      "x-ms-enum": {
+        "name": "TransactionState",
+        "modelAsString": true
+      }
+    },
+    "TransactionStatus": {
+      "description": "Response returned to a query for the transaction status",
+      "required": [
+        "state",
+        "transactionId"
+      ],
+      "properties": {
+        "state": {
+          "$ref": "#/definitions/TransactionState"
+        },
+        "transactionId": {
+          "$ref": "#/definitions/TransactionId"
+        }
+      },
+      "type": "object"
+    },
+    "UserId": {
+      "description": "Identifier for the user. This must either be an AAD object id or a certificate fingerprint.",
+      "type": "string"
+    }
+  },
+  "parameters": {
+    "CommitParameter": {
+      "name": "commit",
+      "in": "query",
+      "required": true,
+      "x-ms-parameter-location": "method",
+      "maximum": 9223372036854775807,
```

Comment:
The `Breaking Change(Cross-Version)` CI check is failing due to [a runtime exception](https://github.com/Azure/azure-rest-api-specs/pull/21659/checks?check_run_id=9712102995) when parsing this value. The value wasn't changed in the new API version and it is exactly the same as the previous one (see [here](https://github.com/Azure/azure-rest-api-specs/blob/1fefe3f5cee88319b17c08a2dbf95e1e983a9f8c/specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/stable/2022-05-13/confidentialledger.json#L922) for comparison). I raised an issue in the `openapi-diff` repository: https://github.com/Azure/openapi-diff/issues/258.

Reply:
@mikekistler @markweitzel @JeffreyRichter @johanste @lynshi Do you have any opinion on the above problem and how we could solve it?

Reply:
I believe it should not be a problem to reduce the number (@lynshi please correct me otherwise). I tried changing the limits to the respective maximum (2^53 – 1) and minimum (-(2^53 – 1)) safe integers in JavaScript, but we are still getting the same runtime exception. If that is ok for the board, we can ignore this error and merge the PR anyway since this is a non-issue in my opinion.